src/coordination/include/coordination/replication_instance.hpp

```diff
@@ -62,7 +66,8 @@ class ReplicationInstance {
   auto SendSwapAndUpdateUUID(const utils::UUID &new_main_uuid) -> bool;
   auto SendUnregisterReplicaRpc(std::string const &instance_name) -> bool;
 
-  // TODO: (andi) Inconsistent API
+
+  auto SendGetInstanceUUID() -> utils::BasicResult<coordination::GetInstanceUUIDError, std::optional<utils::UUID>>;
```

Comment:
in general, what is the point of having here BasicResult when you are not doing anything specific with any error type

Reply:
Just use optional...

Reply:
I answered before, can you check, if it doesn't make sense still I can change it

Reply:
Resolved on call